README.md

```diff
@@ -24,32 +24,53 @@ steps:
 ```
 
 ## Using the plugin
-Build the container:
+
+Get a maintained build from
+[quay.io/arcalot/arcaflow-plugin-pcp](https://quay.io/repository/arcalot/arcaflow-plugin-pcp),
+or build the container locally:
 ```
-docker build . -t arcaflow-plugin-pcp
+podman build . -t arcaflow-plugin-pcp
 ```
 
 Run with the provided example input:
 ```
-cat configs/pcp_example.yaml | docker run -i --rm arcaflow-plugin-pcp -f -
+podman run -i --rm arcaflow-plugin-pcp -s start-pcp -f - < configs/pcp_example.yaml
 ```
 
-***Note:** This plugin is designed to be used as a container image built with
-the provided Dockerfile. Using the python directly on a target system will
-likely prove problematic*
+> [!NOTE]
+> This plugin is designed to be used as a container image built with the provided
+> Dockerfile. Using the python directly on a target system will likely prove problematic
+
+### Post-Processing Only
+
+The `post-process` plugin step can be run independently to convert an existing PCP
+archive file into the same structured format that the main `start-pcp` plugin step
+provides. You will need to provde the path to the existing archive file as the
```

Comment:
I was thinking about changing this, but was biased against a possible breaking change. That said, since the plugin had only one step before, I believe all existing references don't mention the step at all, so all of those references will be "broken" anyway with the new version since they will require the step reference now.